src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -164,6 +164,7 @@
   private static final String VALID_NAME_REGEX = "[\\w]+";
   private static final String INVALID_NAME_REGEX = "[^\\w]+";
   private static final Gson GSON = new Gson();
+  private static final String RETAIN_STAGING_TABLE = "table.staging.retain";
```

Comment:
nit: `"retain.staging.table"` similar to how variable is named.

Reply:
done :)